pkg/federation/sdlmerge/interface_type_extending.go

```diff
@@ -47,3 +58,25 @@ func (e *extendInterfaceTypeDefinitionVisitor) EnterInterfaceTypeExtension(ref i
 		e.Walker.StopWithExternalErr(operationreport.ErrExtensionOrphansMustResolveInSupergraph(e.document.InterfaceTypeExtensionNameBytes(ref)))
 	}
 }
+
+func (e extendInterfaceTypeDefinitionVisitor) getWalker() *astvisitor.Walker {
```

Comment:
this function seems to be redundant, as I could just access the Walker with `e.Walker`, also try to avoid the prefix`get` if possible

Reply:
I tried to use walker() and document(), but there was already a method called document(). I prefixed both with get for consistency.

Regarding e.Walker, this method was just for the interface to work. I didn't know how else to get access to the Walker/document without passing it in.